File: 2021/6/druga.py
def main():
    input = open("input.txt")
    nums = [int(x) for x in input.readline().strip().split(",")]

    slovar = {0: 0, 1: 0, 2: 0, 3: 0, 4: 0, 5: 0, 6: 0, 7: 0, 8: 0}
    for num in nums:
        slovar[num] += 1
    
    stDni = 256
    slovar[8] = 0
    for i in range(stDni):
        novSlovar = {0: 0, 1: 0, 2: 0, 3: 0, 4: 0, 5: 0, 6: 0, 7: 0, 8: 0}
        keys = slovar.keys()
        for key in keys:
            stRib = slovar[key]
            key -= 1
            if (key == -1):
                novSlovar[8] += stRib
                novSlovar[6] += stRib
            else:
                novSlovar[key] += stRib
        slovar = novSlovar
    print(sum(slovar.values()))

main()

File: 2021/6/druga.java
import java.io.BufferedReader;
import java.io.FileReader;
import java.io.IOException;
import java.io.Reader;
import java.util.ArrayList;

public class druga {
    public static void main(String[] args) throws IOException{
        BufferedReader reader = new BufferedReader(new FileReader("input.txt"));
        String[] zacetek = reader.readLine().strip().split(",");
        ArrayList<Integer> zacetek2 = new ArrayList<>();
        for (int i = 0; i < zacetek.length; i++)
            zacetek2.add(Integer.parseInt(zacetek[i]));
        
        for (int j = 0; j < 256; j++) {
            for (int k = 0; k < zacetek2.size(); k++) {

            }

        }
        
    }
}